executor.go

```diff
@@ -594,6 +602,9 @@ func completeValueCatchingError(eCtx *ExecutionContext, returnType Type, fieldAS
 			}
 			if err, ok := r.(gqlerrors.FormattedError); ok {
 				eCtx.Errors = append(eCtx.Errors, err)
+				eCtx.PanicHandler(err)
+			} else {
+				eCtx.PanicHandler(nil)
```

Comment:
Should you add another branch in case r is an error that is not a gqlerrors.FormattedError and call the PanicHandler with that instead of dropping directly to nil?